src/renderers/dom/server/ReactServerUpdateQueue.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactServerUpdateQueue
+ * @flow
+ */
+
+'use strict';
+
+var ReactUpdateQueue = require('ReactUpdateQueue');
+var Transaction = require('Transaction');
+var ReactClass = require('ReactClass');
+var warning = require('warning');
+
+/**
+ * This is the server update queue. It uses ReactUpdateQueue while mounting
+ * then switches to ReactNoopUpdateQueue when mounted
+ * @class ReactServerUpdateQueue
+ * @param {boolean} transaction
```

Comment:
That's the question I myself has on my projects, flow annotations are ok, IF you don't use any doc generation...